### ISSUE TITLE
[Bug]: It does not work from remote containers

### ISSUE THREAD

**Issue by ddnexus** [rxliuli/joplin-utils#50]
### Confirmation

- [X] I checked [FAQ](https://joplin-utils.rxliuli.com/joplin-vscode-plugin/guide/faq.html)

### The tool/kit where the error occurred

joplin-vscode-plugin

### Describe the bug

It does not work in remote containers.

### Steps to reproduce the behavior

- Create a remote container
- Install the plugin in remote container (it does so automatically if you are in the container when you first install)
- You can verify that the url/port is not recognized.

### log file

Not necessary

### Additional context

It requires to be installed in the remote container while it should not.
Installing it that way can work only if you change the container `network_mode` to `"host"`, which is unnecessary and annoying, because you have to add that setting to all the containers and rebuild each one of them.

The work around is overriding its `extensionKind` setting:

```json
{
    "remote.extensionKind": {
        "rxliuli.joplin-vscode-plugin":"ui"
    },
}
```
That way it works as expected from the local host and you don't need to modify anything in your remote containers setup.

However the actual fix is changing its extension kind in the source, so it will work without a hitch without any overwriting workaround.

**Comment by rxliuli** [comment]
Confirmed and fixed, can you try pre-release v0.7.12?

https://github.com/rxliuli/joplin-utils/releases/tag/joplin-vscode-plugin%400.7.12

**Comment by ddnexus** [comment]
Yup, it works! Thank you for the fast reaction!
